isisd/isisd.c

```diff
@@ -93,7 +108,6 @@ void isis_new(unsigned long process_id, vrf_id_t vrf_id)
 	/*
 	 * uncomment the next line for full debugs
 	 */
-	/* isis->debugs = 0xFFFF; */
```

Comment:
nit: please remove the comment above as well.

Reply:
> nit: please remove the comment above as well.
It is removed and squashed into single commit.

Thanks,